packages/react/types/tests.tsx

```diff
@@ -181,6 +182,46 @@ const anim1 = keyframes`
   // https://github.com/DefinitelyTyped/DefinitelyTyped/issues/40993
   // this is really problematic behaviour by @types/react IMO
   // but it's what @types/react does so let's not break it.
-  const CompWithImplicitChildren: React.FC = () => null;
-  ;<CompWithImplicitChildren>content<div/></CompWithImplicitChildren>
+  const CompWithImplicitChildren: React.FC = () => null
+  ;<CompWithImplicitChildren>
+    content
+    <div />
+  </CompWithImplicitChildren>
+}
+
+// Tests for WithConditionalCSSProp
+{
```

Comment:
Im reviewing from mobile and everything is squeezed so maybe im seeing this wrong but it seems that those tests only compute types. Couldwe rewrite them in a way that they would act as assertions? So tests would a tually fail if we introduce a regression? Such test could be written by comparing the output to the expected type

Reply:
Good point. I wrote hoping they'll crash if there's a regression, but any `any` in LibraryManageAttributes would wreak them. I can certainly improve them with $ExpectType.